app/src/main/res/values/dimens.xml

```diff
@@ -26,4 +26,27 @@
     <dimen name="child_program_linear_layout_margin">8dp</dimen>
     <dimen name="launcher_btn_height">80dp</dimen>
     <dimen name="launcher_btn_width">0dp</dimen>
+    <dimen name="checkbox_margin_top">8dp</dimen>
+    <dimen name="device_status_icon_width">10dp</dimen>
+    <dimen name="margin_status_icon">16dp</dimen>
+    <dimen name="home_set_up_card_bg">16dp</dimen>
+    <dimen name="match_constraint_width">0dp</dimen>
+    <dimen name="match_constraint_height">0dp</dimen>
+    <dimen name="home_card_one_corner_radius">4dp</dimen>
+    <dimen name="home_card_one_elevation">3dp</dimen>
+    <dimen name="home_card_one_margin">32dp</dimen>
+    <dimen name="home_card_one_margin_top">8dp</dimen>
+    <dimen name="margin_right_end_device_status_textview">8dp</dimen>
+    <dimen name="margin_home_checkboxes">16dp</dimen>
+    <dimen name="padding_home_checkboxes">20dp</dimen>
+    <dimen name="margin_top_home_checkboxes">8dp</dimen>
+    <dimen name="home_card_two_elevation">4dp</dimen>
+    <dimen name="margin_set_up_title">8dp</dimen>
+    <dimen name="margin_top_set_up_title">24dp</dimen>
+    <dimen name="height_seperator_horizontal">1dp</dimen>
+    <dimen name="width_seperator_vertical">1dp</dimen>
+    <dimen name="height_seperator_vertical">150dp</dimen>
+    <dimen name="margin_seperator_vertical">8dp</dimen>
+    <dimen name="btn_vj_margin_top">16dp</dimen>
+    <dimen name="btn_vj_margin_right_end">32dp</dimen>
```

Comment:
Same, use standard set of dimens, no need to duplicate

Reply:
These are the dimens value for different attributes and they can have duplicate values, Suppose in future if someone changes the value of particular **_dimen_** then it will affect other uses too.

Reply:
Then in future, we'll add a separate dimen

Reply:
Okay, now what to do, May I know Why aren't you merging the PR then?

Reply:
Because "Same, use standard set of dimens, no need to duplicate"